src/components/buttons/play-button.ts

```diff
@@ -1,4 +1,4 @@
-import { PLAY_BUTTON } from "@constants";
+import { PLAY_BTN_IMG } from '@constants'
```

Comment:
lets use double quotes just to be consistent for imports

Reply:
Changes applied, thank you.